kernelci/build.py

```diff
@@ -450,15 +450,20 @@ def _run_make(kdir, arch, target=None, jopt=None, silent=True, cc='gcc',
     return shell_cmd(cmd, True)
 
 
-def _make_defconfig(defconfig, kwargs, fragments, verbose, log_file):
+def _make_defconfig(defconfig, kwargs, fragments, verbose, log_file, opts):
```

Comment:
No need to pass `opts` since a copy is made in the function.

(hrm, I decline all liability for this one ;-) )